com.longbridgeapp.LongbridgePro.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.longbridgeapp.LongbridgePro
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+command: longbridgepro
+
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+
+modules:
+  - name: longbridgepro
+    buildsystem: simple
+    build-commands:
+      - install -D longbridgepro.sh /app/bin/longbridgepro
+      - |
+        for s in 16 32 48 64 128 256 512; do
+          install -Dm644 longbridgepro_${s}.png ${FLATPAK_DEST}/share/icons/hicolor/${s}x${s}/apps/com.longbridgeapp.LongbridgePro.png
+        done
+      - install -Dm644 com.longbridgeapp.LongbridgePro.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 com.longbridgeapp.LongbridgePro.desktop -t /app/share/applications
+      - install -D apply_extra -t /app/bin
+    sources:
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - |
+            bsdtar -Oxf longbridgepro.deb 'data.tar.xz' |
+              bsdtar -xf -
+          - mv opt/Longbridge\ Pro longbridgepro
+          - rm -r longbridgepro.deb opt usr
+      - type: file
+        path: com.longbridgeapp.LongbridgePro.desktop
+      - type: file
+        path: com.longbridgeapp.LongbridgePro.metainfo.xml
+      - type: file
+        path: longbridgepro.sh
+      - type: dir
+        path: icons
+
+      - type: extra-data
+        filename: longbridgepro.deb
+        only-arches: [x86_64]
+        url: https://download.lbkrs.com/longbridge-desktop/prod/longbridge-pro-1.7.3-latest-linux-amd64.deb
+        sha256: 0eb0b6fe90fb7041ada5cdf9c956dc8566f5fdf6b4aab4aceccb32278d759b7a
+        size: 93106690
+
+      - type: extra-data
+        filename: longbridgepro.deb
+        only-arches: [aarch64]
+        url: https://download.lbkrs.com/longbridge-desktop/prod/longbridge-pro-1.7.3-latest-linux-arm64.deb
+        sha256: 5ccd41c64ac2e06e3e87c9de59cfacd465bec7ef01b4cd9b22a3c6b2b19106ab
+        size:  90131226
+
+  - name: openssl11
```

Comment:
so let me get this straight, this use unmaintained version of the crypto libraries. OK.

Reply:
Yeah, the application require openssl 1.1 to work.